### ISSUE TITLE
SQL Server Side Limitation

### ISSUE THREAD

**Issue by ocinbat** [dotnet/EntityFramework.Docs#2240]
It's worth mentioning that we won't be able to see those queries on sys.dm_exec_requests statement_text because comments are stripped.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4ff9c2c3-163b-0d1d-e3b0-b7351f39592e
* Version Independent ID: 35b7f4ad-ea1c-d42e-d601-8b56f418a194
* Content: [Query Tags - EF Core](https://docs.microsoft.com/en-us/ef/core/querying/tags#feedback)
* Content Source: [entity-framework/core/querying/tags.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/core/querying/tags.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @divega
* Microsoft Alias: **avickers**

**Comment by ajcvickers** [comment]
@ocinbat Yes, we should document this. See the discussion here: https://github.com/dotnet/efcore/issues/20105
